syntax = "proto3";

package proto;

service SendBinary {
	rpc sendBinary(stream input) returns (stream output);
}

message input {
	bytes file = 1;
	int32 num_chunk = 2;
}

message output {
	string antivirus = 1;
	string response = 2;
	int32 num_chunk = 3;
}
